### PR TITLE
feat: load scripts conditionally based on path 

### PR DESCRIPTION
**Pull request by ahmaxed**
- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] None of my changes are plagiarized from another source without proper attribution.
- [x] All the files I changed are in the same world language (for example: only English changes, or only Chinese changes, etc.)
- [x] My changes do not use shortened URLs or affiliate links.

Closes #XXXXX

### CODE REVIEW

**Review by raisedadead — APPROVED** [review]

File: client/utils/gatsby/layoutSelector.js [hunk 1]
```diff
@@ -7,6 +7,19 @@ import {
 } from '../../src/components/layouts';
 import FourOhFourPage from '../../src/pages/404';
 
+const scriptAdd = (id, key, async, src) => {
+  var s = document.createElement('script');
```
Comment (ojeytonwilliams) [inline client/utils/gatsby/layoutSelector.js]: I think this need to change as `document` doesn't exist during SSR.

Comment (ojeytonwilliams) [reply client/utils/gatsby/layoutSelector.js]: @raisedadead I've double-checked this and I get
DocumentSSR
when I try to build. I don't understand how [this build](https://dev.azure.com/freeCodeCamp-org/freeCodeCamp/_build/results?buildId=220&view=results) succeeded.

Comment (raisedadead) [reply client/utils/gatsby/layoutSelector.js]: Thanks for catching this, I have reverted this PR.

**Comment by ojeytonwilliams** [comment]
Also, from the contributor's chat:

the donation modal still relies on Stripe.

File: client/utils/gatsby/layoutSelector.js (hunk 1 above)
Comment (ahmadabdolsaheb) [reply client/utils/gatsby/layoutSelector.js]: Thanks for catching this.
I will find another way.
Use helmet perhaps.

**Pull request merged**